etc/tikv-importer.toml

```diff
@@ -49,3 +49,5 @@ num-import-jobs = 24
 # region-split-size = "96MB"
 # stream channel window size, stream will be blocked on channel full.
 # stream-channel-window = 128
+# maximum number of open engines
+max-open-engines = 6
```

Comment:
Could the max-open-engines be increased to 8? Or should we reduce the table-concurrency in lightning to 6?

cc @holys